sdk/core/amqp-common/src/errors.ts

```diff
@@ -526,6 +527,19 @@ export function isSystemError(err: any): boolean {
   return result;
 }
 
+export function isWebsocketError(err: any): boolean {
+  let result: boolean = false;
+  if (
+    !isNode &&
+    window !== undefined &&
```

Comment:
I think this check is superfluous given what isNode does?

Reply:
I had copy pasted this based on offline discussion with @ramya-rao-a 
I believe this is on purpose as we'll need another issue/item to update all isNode references, and likely move it out to a separate module.